src/aleph_client/commands/instance/superfluid.py

```diff
@@ -57,7 +56,8 @@ async def update_flow(account: ETHAccount, receiver: str, flow: Decimal, update_
         if current_flow_rate_wei > 0:
             # Reduce the existing flow
             new_flow_rate_wei = current_flow_rate_wei - flow_rate_wei
-            if new_flow_rate_wei > 0:
+            # Ensure to not leave infinitesimal flows
+            if new_flow_rate_wei > 99:
```

Comment:
That looks very arbitrary :thinking:

Reply:
We have to do this to avoid rounding errors on amounts. Most of flows have 1-10 wei left after being decreased, which is blocking their deletion. 100 wei is a safe limit and 100 wei = 1e-16 eth  = basically nothing